pyvista/core/composite.py

```diff
@@ -190,7 +190,7 @@ def bounds(self) -> BoundsLike:
         ... ]
         >>> blocks = pv.MultiBlock(data)
         >>> blocks.bounds
-        (-0.5, 2.5, -0.5, 2.5, -0.5, 0.5)
+        (np.float64(-0.5), np.float64(2.5), np.float64(-0.5), np.float64(2.5), np.float64(-0.5), np.float64(0.5))
```

Comment:
In #6271, I supported following the numpy data types, but your point made me realize that the more important thing here is that the data type is a tuple of floats. I created a patch for this PR at #6282, so please check it out.

Reply:
Let's keep the work here as it is likely that one decision for one part will trickle over into other parts. 

> How annoying do you think this repr change is? We're going out of our way to return a tuple instead of a numpy array in `.bounds`, so we could just explicitly return a tuple of native floats. 

I think this whole function could be simplified to directly form a tuple rather than doing this to `numpy.stack` and `ravel` magic here.


> And fix the regular mesh .bounds as well.

This needs more digging.  This typing `cast` makes me suspicious about the `DataSet.bounds` method too.  It might just be a length of container thing, but it is worth aligning these methods.

https://github.com/pyvista/pyvista/blob/c6ee1a6cc9d56774f2140007455e70e7b1f75318/pyvista/core/dataset.py#L1752

Reply:
I separate #6282 as other PR. Please go on as is here.

Reply:
I am using something different than #6282 now.  Take a look please @tkoyama010